nilearn/decomposition/_base.py

```diff
@@ -274,10 +276,12 @@ class _BaseDecomposition(CacheMixin, TransformerMixin, BaseEstimator):
            :obj:`~nilearn.surface.SurfaceImage` or
            :obj:`~nilearn.maskers.SurfaceMasker` object, optional
         Mask to be used on data. If an instance of masker is passed,
-        then its mask will be used. If no mask is given, it will be computed
-        automatically by a MultiNiftiMasker for Niimg-like objects with default
-        parameters and no mask will be used for SurfaceImage objects.
+        then its mask will be used. If no mask is given, for Nifti images,
+        it will be computed automatically by a MultiNiftiMasker with default
+        parameters; for surface images, all the vertices will be used.
```

Comment:
This isn't exactly true as of yet.
- See: https://github.com/nilearn/nilearn/issues/5390. Will fix in a separate PR